src/fidesops/main.py

```diff
@@ -172,6 +174,10 @@ def start_webserver() -> None:
         )
     )
 
+    if not config.execution.USE_DEDICATED_WORKER:
+        logger.info("Starting worker...")
+        subprocess.Popen(["fidesops", "worker"])
```

Comment:
There may well be a smarter way to run this on the same container. For instance this wouldn't be any more performant than Fidesops prior to Celery if this was run outside of a high availability environment, as the same server is doing the work albeit in a slightly different structure. The main benefits of this are:
- giving users the option to run everything with as small container footprint as possible
- saving developers time and memory building + operating the worker container to test the privacy request flow locally